.eslintrc.json

```diff
@@ -25,7 +25,6 @@
     "packages/*/lib",
     "!.storybook",
     ".storybook/public",
-    "src/app/_components/custom-constellation",
     "src/app/_components/custom-sdk/**/demo.stories.*",
     "src/app/_components/custom-sdk/**/mock.stories.*",
```

Comment:
Shouldn't we remove `stories` from `mock.stories.*` since we got rid of stories from the mocks?

Reply:
Hi Vishal, please approve so I can merge